docker/anaconda3/Dockerfile

```diff
@@ -1,7 +1,10 @@
 # docker build --tag alleninstitute/allensdk:anaconda3 .
 # docker run -it alleninstitute/allensdk:anaconda3 /bin/bash
 #
-FROM continuumio/miniconda3:4.8.2
+# To deploy an image for use on bamboo, use the following command:
```

Comment:
is the above not relevant then?

Reply:
I think that's correct. I'll remove the two of them.